<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XSDEnumDialog</class>
 <widget class="QDialog" name="XSDEnumDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit XSD Enumeration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../risorse.qrc">
    <normaloff>:/icon/images/icon.png</normaloff>:/icon/images/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Facets (enumerations/restrictions)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableWidget" name="facets"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="cmdAdd">
           <property name="text">
            <string>Add...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cmdMod">
           <property name="text">
            <string>Modify</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cmdAnnot">
           <property name="text">
            <string>Annotation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cmdDel">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cmdMoveUp">
           <property name="text">
            <string>Move Up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cmdMoveDown">
           <property name="text">
            <string>Move Down</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../risorse.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>XSDEnumDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>XSDEnumDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
